pipelines/components/transform_android.py

```diff
@@ -9,42 +9,49 @@ def bigquery_raw_data_to_typed_data_op(raw_data_table: str, updates_table: str,
     def transform_raw_data_to_typed_data_query():
         android_fields = [
             # Regular fields
-            {"name": "stats_timestamp", "from_name": "Stats_timestamp"},
-            {"name": "stats_version", "from_name": "Stats_Model_Version"},
-            {"name": "app_bundle_identifier", "from_name": "applicationId"},
+            {"name": "stats_timestamp", "from_name": "Stats_timestamp", "hash": False},
+            {"name": "stats_version", "from_name": "Stats_Model_Version", "hash": True},
+            {"name": "stats_library_version", "from_name": "Stats_Version", "hash": False},
+            {"name": "app_bundle_identifier", "from_name": "applicationId", "hash": True},
             # Fields w.r.t. system
-            {"name": "system_sdk_version", "from_name": "sdkVersion"},
-            {"name": "system_default_language", "from_name": "defaultLanguage"},
-            {"name": "manufacturer", "from_name": "manufacturer"},
-            {"name": "model_name", "from_name": "modelName"},
+            {"name": "system_sdk_version", "from_name": "sdkVersion", "hash": True},
+            {"name": "system_default_language", "from_name": "defaultLanguage", "hash": True},
+            {"name": "manufacturer", "from_name": "manufacturer", "hash": True},
+            {"name": "model_name", "from_name": "modelName", "hash": True},
             # Fields w.r.t. preference
-            {"name": "preference_daytime", "from_name": "daytime"},
-            {"name": "preference_is_nightmode_enabled", "from_name": "isNightmodeEnabled", "type": "BOOLEAN"},
+            {"name": "preference_daytime", "from_name": "daytime", "hash": False},
```

Comment:
Is this intentional, we included this one in the hash before, right?

Reply:
Good catch, wasn't intentional